packages/cli/src/commands/validator/forceDeaffiliate.ts

```diff
@@ -0,0 +1,36 @@
+import { BaseCommand } from '../../base'
+import { newCheckBuilder } from '../../utils/checks'
+import { displaySendTx } from '../../utils/cli'
+import { Flags } from '../../utils/command'
+
+export default class ValidatorForceDeaffiliate extends BaseCommand {
+  static description =
+    'Force deaffiliate a Validator from a Validator Group, and remove it from the Group if it is also a member.'
```

Comment:
add: Used by stake-off admins in order to remove validators from the next epoch's validator set if they are down and consistently unresponsive, in order to preserve the health of the network. This feature will be removed once slashing for downtime is implemented.